course/src/Vector3.cc

```diff
@@ -0,0 +1,211 @@
+#include "Vector3.h"
+
+namespace ekumen
+{
+
+    Vector3::Vector3(const double x, const double y, const double z)
+    { 
+        values = new double[3];        
+        values[0] = x;
+        values[1] = y;
+        values[2] = z;
+    }
+
+    Vector3::Vector3()
+    {   
+        values = new double[3];
+        values[0] = 0.;
+        values[1] = 0.;
+        values[2] = 0.;
+    }
+
+    Vector3::Vector3(const std::initializer_list<double> &list)
+    {   
+        values = new double[3];
+        if (list.size() == 3)
+        {
+            auto i = 0;            
+            for (const auto &v : list)
+            {   
+                values[i] = v;                
+                ++i;
+            }
+            
+        }
+    }
+
+    Vector3::Vector3(const Vector3 &p2)
+    {   
+        values = new double[3];
```

Comment:
Remember to free the memory that was previously pointed to by `values`, or you'll leak memory!

Reply:
it was fixed in previous PR.
    Vector3::~Vector3(){
        delete[] values;
    }